<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddMembeForm</class>
 <widget class="QDialog" name="AddMembeForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>836</width>
    <height>737</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="title_label">
     <property name="styleSheet">
      <string notr="true">font: 75 16pt &quot;MS Shell Dlg 2&quot;;
color: rgb(85, 170, 255)</string>
     </property>
     <property name="text">
      <string>Pakistan Lawers Federation (PLF)</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;Calibri&quot;;</string>
         </property>
         <property name="text">
          <string>Registration Number#</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;Calibri&quot;;</string>
         </property>
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;Calibri&quot;;</string>
         </property>
         <property name="text">
          <string>Father</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;Calibri&quot;;</string>
         </property>
         <property name="text">
          <string>CNIC</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;Calibri&quot;;</string>
         </property>
         <property name="text">
          <string>Permanent Contact</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;Calibri&quot;;</string>
         </property>
         <property name="text">
          <string>Emergeny Contact</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;Calibri&quot;;</string>
         </property>
         <property name="text">
          <string>Permanent Address</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;Calibri&quot;;</string>
         </property>
         <property name="text">
          <string>Temporary Address</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;Calibri&quot;;</string>
         </property>
         <property name="text">
          <string>City</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_10">
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;Calibri&quot;;</string>
         </property>
         <property name="text">
          <string>Blood Group</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_11">
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;Calibri&quot;;</string>
         </property>
         <property name="text">
          <string>Merital Status</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_12">
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;Calibri&quot;;</string>
         </property>
         <property name="text">
          <string>Guardian of Member (After Death)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_13">
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;Calibri&quot;;</string>
         </property>
         <property name="text">
          <string>Relation with member</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_14">
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;Calibri&quot;;</string>
         </property>
         <property name="text">
          <string>CNIC No. of Gaurdian</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_15">
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;Calibri&quot;;</string>
         </property>
         <property name="text">
          <string>Legal Study Status</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_16">
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;Calibri&quot;;</string>
         </property>
         <property name="text">
          <string>Year Of Study</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_17">
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;Calibri&quot;;</string>
         </property>
         <property name="text">
          <string>Institution name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_18">
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;Calibri&quot;;</string>
         </property>
         <property name="text">
          <string>Session of Degree</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_22">
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;Calibri&quot;;</string>
         </property>
         <property name="text">
          <string>License Issuing bar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_21">
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;Calibri&quot;;</string>
         </property>
         <property name="text">
          <string>License Number</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_19">
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;Calibri&quot;;</string>
         </property>
         <property name="text">
          <string>City of practice</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_20">
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;Calibri&quot;;</string>
         </property>
         <property name="text">
          <string>Court of practic</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QSpinBox" name="registration_number_spinbox"/>
       </item>
       <item>
        <widget class="QLineEdit" name="name_lineedit"/>
       </item>
       <item>
        <widget class="QLineEdit" name="father_name_lineedit"/>
       </item>
       <item>
        <widget class="QLineEdit" name="cnic_lineedit"/>
       </item>
       <item>
        <widget class="QLineEdit" name="permanent_contact"/>
       </item>
       <item>
        <widget class="QLineEdit" name="emergency_contact"/>
       </item>
       <item>
        <widget class="QLineEdit" name="permanent_address_lineedit"/>
       </item>
       <item>
        <widget class="QLineEdit" name="temporary_address"/>
       </item>
       <item>
        <widget class="QLineEdit" name="city_lineedit"/>
       </item>
       <item>
        <widget class="QLineEdit" name="bloodgroup"/>
       </item>
       <item>
        <widget class="QComboBox" name="merital_status"/>
       </item>
       <item>
        <widget class="QLineEdit" name="guradian_name"/>
       </item>
       <item>
        <widget class="QLineEdit" name="relation_with_member"/>
       </item>
       <item>
        <widget class="QLineEdit" name="cnic_of_guardian"/>
       </item>
       <item>
        <widget class="QLineEdit" name="legal_study_status"/>
       </item>
       <item>
        <widget class="QLineEdit" name="year_of_study"/>
       </item>
       <item>
        <widget class="QLineEdit" name="institute_name"/>
       </item>
       <item>
        <widget class="QLineEdit" name="session_of_degree"/>
       </item>
       <item>
        <widget class="QLineEdit" name="license_issuing_bar"/>
       </item>
       <item>
        <widget class="QLineEdit" name="license_number"/>
       </item>
       <item>
        <widget class="QLineEdit" name="city_of_practice"/>
       </item>
       <item>
        <widget class="QLineEdit" name="court_of_practice"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Add Member</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
